### ISSUE TITLE
Add 'Market Price' and 'Deviation' to all Offer IDs and Trade IDs contained within Portfolio and Support sections

### ISSUE THREAD

**Issue by pazza83** [bisq-network/bisq#5192]
### Description

Currently it is difficult on Bisq for traders to know what the Market Price was at the time their offers were taken.

Market price is dynamic yet the information is not captured in the Trade or Offer IDs. 

It is also difficult on Bisq for traders to know what deviation from market price was used on the trades that were accepted

The absence of this information makes certain information difficult to ascertain quickly this includes:

- Profit / loss
- If the trade amounts shown are accurate
- Mediator proposed suggestions for trade penalties
- What the market price was at the time of trade
- What the deviation was at the time of trade

### Solution

Add to the Offer IDs and Trade IDs the following information

- Market price
- Deviation

The market price shown would be the current market price in the case of Offer IDs. 

The market price shown would be the market price at the time of trade in the case of Trade IDs. 

This information could be added to the following sections:

**Portfolio**

- My open offers
- Open trades
- History
- Failed

**Support**

- Mediation 
- Arbitration 

#### Version

v1.5.5

### Steps to reproduce

Click on Trade ID in the support section. You will not be able to see deviation from market price or BTC price at the time of trade.

Click on Trade ID in the my portfolio section. You will not be able to see deviation from market price or BTC price at the time of trade.

Click on Offer ID in the my open offers section. You will not be able to see deviation from market price or BTC price within the offer ID.

### Expected behaviour

Click on Trade ID in the support section and be able to see deviation from market price or BTC price at the time of trade.

Click on Trade ID in the my portfolio section and able to see deviation from market price or BTC price at the time of trade.

Click on Offer ID in the my open offers section and able to see deviation from market price or BTC price within the offer ID.

### Screenshots

![dispute details](https://user-images.githubusercontent.com/66367547/107863496-c1078f00-6e4c-11eb-8ae6-a2169adae8a0.png)

Example taken showing a disputed trade ID in the Support > Mediation section

![dispute details (proposed)](https://user-images.githubusercontent.com/66367547/107863508-d8467c80-6e4c-11eb-94b1-3660f5486f35.png)

Example of how I would proposed a disputed trade ID appearing in the Support > Mediation section

**Comment by Conza88** [comment]
Yes please! Should be making it as easy as possible to export and determine overall costs (mining fees + trading fees) etc. and profit/loss for trades. See how well portfolio is performing. I had to manually add deviations column. 

Also currencies column would help possibly... when buying/selling in variety of currencies e.g. "100 EUR" in the text field, when others "300 USD" etc. have to remove all "EUR/USD/CAD" text in the export, then try convert back to home/default/one currency manually. 

Related: #5069 [Add a further column/s to make "Portfolio Performance" easier to determine](https://github.com/bisq-network/bisq/issues/5069)

**Comment by pazza83** [comment]
Hi are there are @bisq-network/bisq-devs able to take this on?

**Comment by wallclockbuilder** [comment]
Ready. Waiting for approval.

**Comment by sqrrm** [comment]
Makes sense to me and there's support for this issue, assigned it to @wallclockbuilder

**Comment by pazza83** [comment]
Hi @wallclockbuilder how are you progressing with this issue?

**Comment by wallclockbuilder** [comment]
Working on it.

**Comment by wallclockbuilder** [comment]
@pazza83 I'm working on it.

**Comment by pazza83** [comment]
Hi @xyzmaker123 let me know if you would like to work on this issue.
